code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -64,10 +79,26 @@
 		holy_weapon.reskinned = TRUE
 		M.unEquip(src)
 		M.put_in_active_hand(holy_weapon)
+		if(sanctify_force)
+			holy_weapon.sanctify_force = sanctify_force
+			holy_weapon.name = "sanctified " + holy_weapon.name
 		qdel(src)
 
-/obj/item/nullrod/fluff		//fluff subtype to be used for all donator nullrods
+/obj/item/nullrod/afterattack(atom/movable/AM, mob/user, proximity)
+	. = ..()
+	if(!sanctify_force)
+		return
+	if(isliving(AM))
+		var/mob/living/L = AM
+		L.adjustFireLoss(sanctify_force) // Bonus fire damage for sanctified (ERT) versions of nullrod
```

Comment:
Ablaze with holy fire etc.

Reply:
Pretty much. I decided not to have a second message for this, since its an on-hit effect and two messages (this plus the normal one) would potentially spam chat.